bench/wtperf/track.c

```diff
@@ -132,6 +132,11 @@ sum_update_ops(WTPERF *wtperf)
 {
 	return (sum_ops(wtperf, offsetof(WTPERF_THREAD, update)));
 }
+uint64_t
+sum_modify_ops(WTPERF *wtperf)
+{
+	return (sum_ops(wtperf, offsetof(WTPERF_THREAD, modify)));
+}
```

Comment:
Make the location of all of the functions added here alphabetized with the other similar functions.

Reply:
changed according to the other functions. Alphabetized.